kong/tools/queue.lua

```diff
@@ -314,6 +317,45 @@ function Queue.can_enqueue(queue_conf, entry)
   return _can_enqueue(queue, entry)
 end
 
+local function handle(self, entries)
+  local entry_count = #entries
+
+  local start_time = now()
+  local retry_count = 0
+  while true do
+    self:log_debug("passing %d entries to handler", entry_count)
+    local status, ok, err = pcall(self.handler, self.handler_conf, entries)
```

Comment:
Could the handler sends partial entries for each `while true`?

Reply:
Could you explain a little bit?

Reply:
For example, the first loop, sends 1 entry, the second loop would re-send that one.